roles/dhcp-server/templates/local.conf.j2

```diff
@@ -8,3 +8,6 @@ dhcp-boot=pxelinux.0
 dhcp-authoritative
 enable-tftp
 tftp-root=/srv/tftp
+no-hosts
```

Comment:
changing the server's IP in its own /etc/hosts

I went down that path and got angry :p

d-i builds this:
juser@dc10b:~$ cat /etc/hosts
127.0.0.1	localhost
127.0.1.1	dc10b

is ...1.1 needed?

I made a task using  lineinfile but my code kept adding additional "ip \t host" lines each time I re-ran.  so I switched to not using etc/hosts and using a separate file for dnsmasq and was happy.  

I have mixed feelings about dnsmasq using etc/hosts.   

give me the ansible to just solve all of this and I'll give it a whirl